tools/test/examples/update.py

```diff
@@ -17,43 +52,56 @@
 import examples_lib as lib
 from examples_lib import SUPPORTED_TOOLCHAINS
 
-def run_cmd(command, print_warning_on_fail=True):
-    """ Takes the command specified and runs it in a sub-process, obtaining the return code.
-        
-    Args:
-    command - command to run, provided as a list of individual fields which are combined into a 
-              single command before passing to the sub-process call.
-    return_code - result of the command.
+def run_cmd(command, exit_on_failure=False):
+    """ Passes a command to the system and returns a True/False result once the 
+        command has been executed, indicating success/failure. Commands are passed
+        as a list of tokens. 
+        E.g. The command 'git remote -v' would be passed in as ['git', 'remote', '-v']
```

Comment:
We generally have a single sentence summary (SSS) :stuck_out_tongue_closed_eyes: at the beginning of our funcitons, followed by a newline and an expanded description.